MONITORING.md

```diff
@@ -0,0 +1,28 @@
+The `kudo-spark-operator` is able to be seamlessly integrated with Prometheus, which is installed within [Prometheus operator](https://github.com/coreos/prometheus-operator).
+Integration with other Prometheus distributions, like kube-prometheus, wasn't tested. 
+
+The `prometheus-operator` uses crafted services discovery approach, introducing `ServiceMonitor` kind. 
+But the `kudo-spark-operator` take its configuration burden on itself.
+
+#### How to get `kudo-spark-operator` and app's metrics available for scrape by Prometheus:
+1) Ensure `prometheus-operator` is installed on your Kubernetes cluster.
+1) Install the `kudo-spark-operator` with `-p enableMetrics=true` parameter. (But it's true by default, so typically no need to specify it explicitly)
+1) Composing your Spark Application yaml:
+   - use Spark image with JMXPrometheus exporter jar on the board i.e. `gcr.io/spark-operator/spark:v2.4.4-gcs-prometheus` 
```

Comment:
I agree, here should be `mesosphere/spark` image. The reason why I didn't put it here this time - the aforementioned image isn't verified in terms of metrics exporting. I remember you have done good job adding Prometheus configs there, but the image should be assembled, pushed into Docker Hub and checked that the metrics exported. I'm going to do in scope of other PR (introducing Operator dashboard). So, I'd like leave it as is here at this time.

Reply:
> the image should be assembled, pushed into Docker Hub and checked that the metrics exported

yes, this is basically what I've done in scope of testing #12 to verify the metrics are exported into Prometheus. The only difference is that I used my personal docker hub to do that. Since it's a readme file, I don't see any issue with specifying `mesosphere/image` here at the first place just to prevent any confusion or just to not to forget to do it later

Reply:
You can use this image `mesosphere/spark:2.4.4-bin-hadoop2.7-k8s`

Reply:
Thanks @alembiewski. As we discussed, metrics exporting is verified with this image. So it makes sense to mention it here instead of Google's one.